dataflowengine/src/main/scala/io/shiftleft/dataflowengine/language/Path.scala

```diff
@@ -19,7 +19,7 @@ object Path {
 
         val trackedSymbol = trackingPoint match {
           case _: nodes.MethodParameterIn =>
-            val paramsPretty = method.parameter.l.sortBy(_.order).map(_.code).mkString(", ")
+            val paramsPretty = method.parameter.toList.sortBy(_.order).map(_.code).mkString(", ")
```

Comment:
The `.l` shorthand is pretty handy. Wonder whether we should introduce it on top of scala collections at some point. Not in this PR though.